Makefile

```diff
@@ -32,6 +32,8 @@ HAS_GOLANGCI := $(shell ls _output/bin/golangci-lint 2> /dev/null)
 GOFUMPT_VERSION := v0.4.0
 HAS_GOFUMPT := $(shell command -v gofumpt 2> /dev/null)
 
+GO_VERSION := $(shell sed -En 's/^go (.*)$$/\1/p' go.mod)
```

Comment:
I like `go mod edit -json | jq -r .Go`. Though, it introduces a dependency on jq.

Reply:
From https://go.dev/ref/mod#go-mod-file-go:
```
GoDirective = "go" GoVersion newline .
GoVersion = string | ident .  /* valid release version; see above */
```

Hopefully the syntax does not change in the future. In the worst case users will fail to pull the golang image.

Can you test of `jq` existence and run `go mod edit -json | jq -r .Go`. Otherwise, fall back to the current line?

Reply:
seems like unnecessary complexity for a local make command, no?

let me see what other sig projects do

Reply:
no real standard. pushed up a commit to apply your suggestion. let me know what you think